stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/SocketListener.kt

```diff
@@ -12,9 +13,16 @@ public open class SocketListener {
     public open fun onConnected(event: ConnectedEvent) {
     }
 
+    @Deprecated(
+        "Use method with DisconnectCause",
+        replaceWith = ReplaceWith("Use method with DisconnectCause instead of it")
```

Comment:
Let's add level so it will be easier to update it in the future
```suggestion
        replaceWith = ReplaceWith("Use method with DisconnectCause instead of it"),
        level = DeprecationLevel.WARNING,
```